packages/algob/src/lib/tx.ts

```diff
@@ -36,11 +36,25 @@ export async function mkSuggestedParams (
 export function makeAssetCreateTxn (
   name: string, asaDef: ASADef, flags: ASADeploymentFlags, txSuggestedParams: tx.SuggestedParams
 ): tx.Transaction {
+  // Load Note
+  // Load payFlags.note (ignored if payFlags.noteb64 is present)
+  // undefined if none of them is present.
+  // If TxParams has noteb64 or note , it gets precedence
```

Comment:
let's keep only the last one (and remove lines 39-41)

Reply:
ok